index.html

```diff
@@ -197,6 +197,49 @@ <h3 class="frameworks">Skills</h3>
         </div>
     </section>
 
+    <section class="contact-form">
+       <section>
+        <div class="contact-form">
+
+            <p class="footer-text">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+
+            <form class="form-content" action="https://formspree.io/f/mvonwygw" method="POST">
+                <input type="text" name="full_name" maxlength="30" placeholder="Full name" required /><br>
+                <input type="email" name="email" placeholder="Email" required /><br>
+                <textarea name="textarea" maxlength="500" placeholder="Write me something..." required></textarea><br>
+                <button class="submit-button" type="submit" value="submit">Get in touch </button><br>
+            </form>
+
+        </div>
+        <div>
+            <hr class="hr-footer">
+            <ul class="social-icons">
+                <li>
+                    <a href="https://github.com/okemboi/"><img src="/images/dark github.svg" alt="GitHub" /></a>
+                </li>
+                <li><a href="https://www.linkedin.com/in/kiptanuikemboi/"><img src="/images/dark linkedin.svg"
+                            alt="Linkedin" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="/images/dark angelist.svg" alt="Angelist" /></a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/ok_kemboi"><img src="/images/dark twitter.svg" alt="Twitter" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="/images/dark medium.svg" alt="Mediumn" /></a>
+                </li>
+            </ul>
+        </div>
+        </footer>
+
+        <div class="shape">
+            <img src="/images/Shape.png" alt="">
+        </div>
+        </div>
+    </section>
```

Comment:
- [ ] Great job implementing the changes required by the previous reviewer. However, there're few things you need to improve on this milestone. It is required to stick to the figma design as close as possbile. This includes font, color, spacing among others. Your design doesn't yet look like that of figma. Kindly give a closer look and try to produce a replica of figma design.

- [ ] Also, at this moment, all the footer icons are not showing. Kindly attend to that too.

### Your Design
![your-form](https://user-images.githubusercontent.com/26597432/230209042-f0a9d041-cb6e-4eb3-9b37-d801dccb6081.png)

### Figma Design
![figma-form](https://user-images.githubusercontent.com/26597432/230209080-730dd97e-0093-4b1e-bc5f-288bb252e859.png)